package.json

```diff
@@ -78,11 +78,13 @@
     "react": "17.0.2",
     "react-native": "0.66.3",
     "react-native-bootsplash": "^3.2.4",
+    "react-native-custom-tabs": "^0.1.7",
```

Comment:
Need to get rid of this as we are using `react-native-inappbrowser-reborn` instead.